### DISCUSSION TITLE
is it possible to change the default admin

### DISCUSSION THREAD

**Discussion by avshvetz** [drakkan/sftpgo#375] [Q&A] [answered]
Hi @drakkan. I have a question is it is possible to change the default admin user or rewrite it somehow? 
Thanks in advance!

> The default credentials are:
> username: admin
> password: password

https://github.com/drakkan/sftpgo/blob/main/docs/rest-api.md

**Comment by drakkan** [comment, marked as answer]
Hi,

this should work by loading initial data, for example:

```
./sftpgo serve --loaddata-from /tmp/initial.json --loaddata-mode 0
```

where `initial.json` contains a json like the following one:

```
{"users":[],"folders":[],"admins":[{"id":1,"status":1,"username":"admin","password":"newpassword","email":"","permissions":["*"],"filters":{}}],"version":7}
```

**Reply by avshvetz** [reply]
Thanks a lot!

**Reply by drakkan** [reply]
The default admin can now be created after the initial installation

https://github.com/drakkan/sftpgo/commit/f2b93c04020f399e4901499ad2b3804773027502
